.github/workflows/release.yml

```diff
@@ -41,3 +43,13 @@ jobs:
           export ORG_GRADLE_PROJECT_signingInMemoryKey=$(gpg --armor --passphrase $GPG_PASSPHRASE --pinentry-mode=loopback --export-secret-keys $GPG_KEY_NAME signing-key | grep -v '\-\-' | grep -v '^=.' | tr -d '\n')
           shred signing-key
           make release
+      - name: sha256
+        run: sha256sum ./protoc-gen-connect-kotlin/build/libs/protoc-gen-connect-kotlin.jar >> sha256.txt
+      - name: Upload release artifacts in GitHub
```

Comment:
Might want to change this to something like `Publish release` since this will actually create a release

Reply:
Yeah there's two publishing going on here though: the sonatype/maven publish and this github one.

Reply:
Publish GitHub artifacts?

Reply:
Sure, the main thing I'm saying is that it creates a GitHub release too, and then pushes the artifacts with the release